vaadin-date-time-picker-flow-parent/vaadin-date-time-picker-flow-integration-tests/src/test/java/com/vaadin/flow/component/datetimepicker/validation/BinderValidationIT.java

```diff
@@ -73,48 +74,6 @@ public void required_changeValue_assertValidity() {
         assertErrorMessage(REQUIRED_ERROR_MESSAGE);
     }
 
-    @Test
-    public void required_badInput_setDateInputValue_blur_assertValidity() {
-        setInputValue(dateInput, "INVALID");
-        dateInput.sendKeys(Keys.TAB);
-        timeInput.sendKeys(Keys.TAB);
-        assertServerInvalid();
-        assertClientInvalid();
-        assertErrorMessage("");
-    }
-
-    @Test
-    public void required_badInput_setTimeInputValue_blur_assertValidity() {
-        setInputValue(timeInput, "INVALID");
-        timeInput.sendKeys(Keys.TAB);
-        assertServerInvalid();
-        assertClientInvalid();
-        assertErrorMessage("");
-    }
```

Comment:
**note:** Moved these tests to `BasicValidationIT.java` as they aren't really tied to Binder usage.